client/components/Stars/index.sass

```diff
@@ -1,10 +1,25 @@
 @function multiple-box-shadow ($n, $px)
   $value: '#{random($px)}px #{random($px)}px #FFF'
   @for $i from 2 through $n
-    $value: '#{$value} , #{random($px)}px #{random($px)}px #FFF'
+    $value: '#{$value} , #{random($px)}px #{random($px)}px #{random-color()})'
 
   @return unquote($value)
 
+@function random-color ()
+  // min-value can be adjusted to give the stars more or less color.
+  // A lower value means more color saturation.
+  // It should be between 0 and 255.
+  $min-value: 160
```

Comment:
```suggestion
  $min-value: 200
```

Reply:
The saturation was a little distracting when the big stars are colors and max saturated, 200 kept subtle and in the background.